pkg/linear-pools/contracts/silo-linear-pool/SiloExchangeRateModel.sol

```diff
@@ -59,11 +59,10 @@ contract SiloExchangeRateModel {
         return localDeposits.divDown(totalShares);
```

Comment:
I'd add a comment here explaining why this function will always return an 18-decimal rate regardless of the `mainToken`. The reason is because Silo ShareTokens (`wrappedToken`) have the same number of decimals as their underlying assets (`mainToken`), and we use the `FixedPoint` math library here which scales the result to 1e18.

```suggestion
        // Note that the exchange rate returned by this function is always scaled to 18 decimals. All Silo ShareTokens
        // match their respective underlying assets' ERC20 decimals, and Silo uses 18-decimal precision for all
        // internal math including interest rates and fees. We match that behavior using the FixedPoint math library,
        // which will upscale the result of this division to 1e18 decimals.
        // solhint-disable max-line-length
        // token decimals: https://github.com/silo-finance/silo-core-v1/blob/70621494b23d333e67442485d7c743ee6e7d22db/contracts/utils/ShareToken.sol#L44
        // internal precision: https://github.com/silo-finance/silo-core-v1/blob/70621494b23d333e67442485d7c743ee6e7d22db/contracts/lib/Solvency.sol#L39
        // solhint-enable max-line-length
        return localDeposits.divDown(totalShares);
```